### DISCUSSION TITLE
关于幻塔

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#508] [General]
妈的，我已经准备跑了，玩起来太累了，时刻担心自己掉队，还没追赶机制

![A9F22104-AEF8-4163-8E8B-CC18B1888E35](https://user-images.githubusercontent.com/13673702/147081467-022909d3-e0d4-4869-979e-a2a896ae4314.png)

![AF5A9D41-C489-48D9-90E5-1A0B5AFFE639](https://user-images.githubusercontent.com/13673702/147081475-4d076807-2e63-44db-b7af-948c58c7337b.png)

大致总结一下吧

1. 追求强度和原神花钱差不多，但是这游戏你必须追求强度，你当咸鱼那玩啥 mmo ，玩 mmo 就是要攀比，攀比那就难受，武器不抽命座比不过别人
2. 怪太强了，氪佬都打不过，角色没无敌帧，怪又是无限续霸体盾，放连携技、闪避和滞空都会被一巴掌拍下来，一段时间不能攻击，非常丢输出，闪避条还用三四下就空了。打个比方就像用杰洛特打义父，boss 霸体还场地里乱窜，追都追不上
3. 瞎🐔8改氪金道具还没补偿，听说过的仅此一家，改动的原因包括非常规使用带来的巨大优势和数值超过了策划预期
4. 地图很好看，美工很不错，音乐一般，建模拉稀，打击感拉稀，剧情恶心人，我差不多看的主线就是舔狗养成记，但是这游戏做到了米家不会搞的场景平滑切换，不会说屏幕黑一下然后什么东西改了，这一点很好
5. 武器的命座和意志套装效果太离谱了，用脚填的特效和数值，特别是意志套装，这辈子没见过这么夸张的装备，艾萨拉的能量之泉都相形见绌
6. 一次性资源，地图资源有用的都不刷新，例如金币拿完了那再也没有了，我两天前把俩球扔到山下现在还在下面呢，我自己还搬不上来
7. 作为 mmo 没有主城，有同屏最大人数，班吉斯码头作为一个港口城市差不多是藏宝海湾的 1/3 大小，还没有深水区，不知道咋跑船的，唯一算是主城的特殊地图海德加没有可交互的 npc ，去了屁用没有就交个任务

**Comment by Arondight** [comment]
救命啊，我看了一晚上幻塔本地结算可以无限 SL 大法的事，笑不活了

**Reply by mark9804** [reply]
我傍晚看到这瓜的时候愣了一会，确定是我理解的那个意思之后开始笑……上次改内存还是2011年用八门神器改金币，十年过去了，幻塔仿佛宿舍楼一直没修的楼道灯一样温暖又熟悉
